### PR TITLE
Scala Config Changes: Part Deux

### PR DESCRIPTION
**Pull request by cmloegcmluin**
In the previous PR, we agreed upon a ton of changes, and that I would make these in a second wave. I did them all within a day and put them on this branch. Maybe it was in an email or something, but I'm pretty sure I said I would wait to submit this to you until we ironed out the issues with the 28th mina, the 1/121k vs 1/1225k, those two DEfLL exceptions, whether we snap possible comma mean bounds at the Extreme level to tina midpoints, anything arising from my bounds analysis, etc. but I think we should act while this stuff is still fresh-ish in our heads, because we don't know how long those other discussions go, or if they'll spawn more in turn. And also I'm already finding that I want the changes re: the 140th, 93rd, and 75th minas included in the code as I work on the bounds analysis on another branch.

The changes below are:
- fixing up headers
- directed comma names
- inclusion of secondary commas
- arc (or rainbow)
- adding ||\ to the end of each limma-fraction EDO notation
- including 137-EDO
- changes per the 140th mina forum topic
- changes to the 75th mina per the consistent 37-limit forum topic

### CODE REVIEW

File: scala/sag_et.par [hunk 1]
```diff
@@ -4,9 +4,11 @@
 ! s = list of subdivisions for above division using non-native fifth
 !
 n 5 3
+|\
 n 7 4
+|\  ||\
```
Comment (dkeenan7) [inline scala/sag_et.par]: You need to replace "||" with "N" everywhere (double-shafts).

File: scala/sag_12r.par [hunk 2]
```diff
@@ -1,4 +1,4 @@
-! Sagittal 12R notation. Dave Keenan & George Secor. 2006-Mar-22, updated 2020-Apr-14
+! Sagittal 12R (Trojan) notation. Dave Keenan & George Secor. 2006-Mar-22, updated 2020-Jun-14
```
Comment (dkeenan7) [inline scala/sag_12r.par]: I thought we agreed to "12-Relative" rather than "12R".

**Review by dkeenan7 — APPROVED** [review]

File: scala/sag_et.par (hunk 1 above)
Comment (cmloegcmluin) [reply scala/sag_et.par]: Yes! Nice catch. Thank you.

File: scala/sag_12r.par (hunk 2 above)
Comment (cmloegcmluin) [reply scala/sag_12r.par]: Also good catch. Thanks. According to http://forum.sagittal.org/viewtopic.php?p=1594#p1594 it looks like the final form was "12-Relative".

File: scala/sag_desc.par [hunk 3]
```diff
@@ -34,15 +34,15 @@ Zeus' lightning bolt crossed
 the key to the underworld
 |(
 u
-5:7-kleisma (~11:13k) (5120/5103)
+5/7-kleisma (~11/13k) (5120/5103)
```
Comment (dkeenan7) [inline scala/sag_desc.par:37]: The way we use directed comma names in sag_desc.par, depends on the outcome of the ongoing discussion here:
http://forum.sagittal.org/viewtopic.php?f=12&t=492

e.g. we may instead decide to show the downward symbol !( with the description "7/5-kleisma (~13/11k) (5103/5120)"

File: scala/sag_12r.par (hunk 2 above)
Comment (dkeenan7) [reply scala/sag_12r.par]: Agreed. Edited my comment above.

File: scala/sag_desc.par (hunk 3 above)
Comment (cmloegcmluin) [reply scala/sag_desc.par:37]: Yes, understood.

**Pull request merged**